### PR TITLE
add activate license elastic cluster

### PR DESCRIPTION
**Pull request by opstacloud**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -11,4 +11,6 @@
   - include_tasks: elasticsearch_search_config.yml
   - include_tasks: elasticsearch_post.yml
   tags:
-    - elasticsearch-config
\ No newline at end of file
+    - elasticsearch-config
+
+- include_tasks: elasticsearch_activate.yml
```
Comment (winggundamth) [inline tasks/main.yml]: You can put this below include_tasks for faster run

```
   when: elasticsearch_license_check.stat.exists 
```

**Review by winggundamth — CHANGES_REQUESTED** [review]
Check comment

File: tasks/main.yml (hunk 1 above)
Comment (bemillenium) [reply tasks/main.yml]: Ok, Done krub

**Review by winggundamth — APPROVED** [review]
LGTM

**Pull request merged**